validator/build.gradle

```diff
@@ -71,15 +71,15 @@ dependencies {
     implementation 'com.github.awslabs:aws-request-signing-apache-interceptor:b3772780da'
 
     // http client
-    implementation("com.squareup.okhttp3:okhttp:4.9.3")
+    implementation("com.squareup.okhttp3:okhttp:5.0.0-alpha.11")
```

Comment:
do we really need to update this one? it is a new major version that is in alpha state.

Reply:
Nope, I will roll this back. good catch